### ISSUE TITLE
Same fragments got computed twice

### ISSUE THREAD

**Issue by xiki-tempula** [selimsami/qforce#23]
I have a test case of heptane and when I set the setting to

```
[scan]
frag_threshold = 1
```

Technically, I would only need to scan two dihedrals. However, I got three dihedrals to scan.

```
CC_H10C4_a4d6f5f498b2476c6fd0ce965c9cba18~1.inp
CC_H10C4_a4d6f5f498b2476c6fd0ce965c9cba18~2.inp
CC_H8C3_d91b46644317dee9c2b868166c66a18c~1.inp
```

Where `CC_H10C4_a4d6f5f498b2476c6fd0ce965c9cba18~1.inp` and `CC_H10C4_a4d6f5f498b2476c6fd0ce965c9cba18~2.inp` seems to be the same.

please find the files attached.

[heptane.zip](https://github.com/selimsami/qforce/files/7596365/heptane.zip)

**Comment by selimsami** [comment]
Yes this is what I was referring to earlier. I know why it happens (related to the graph matching using capping hydrogens to match) but it will take some work to fix it and test it carefully. But it is high on my priority list.  For now, I would suggest running both.

Thanks for the files. That will be a nice test case.
